src/Symfony/Component/Workflow/Registry.php

```diff
@@ -24,17 +23,11 @@ class Registry
     private $workflows = array();
 
     /**
-     * @param Workflow                        $workflow
-     * @param string|SupportStrategyInterface $supportStrategy
+     * @param Workflow                 $workflow
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $supportStrategy)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
I removed the new type hint, And I added it to the **private** method `supports`